server/pulp/plugins/types/database.py

```diff
@@ -48,18 +48,21 @@ def __str__(self):
         return 'MissingDefinitions [%s]' % ', '.join(self.missing_type_ids)
 
 
-def update_database(definitions, error_on_missing_definitions=False, drop_indices=False):
+def update_database(definitions, error_on_missing_definitions=False, drop_indices=False,
+                    create_indexes=False):
```

Comment:
I'll suggest this should this default to `True` since that was the previous behavior.